packages/renderer/src/lib/statusbar/StatusBar.svelte

```diff
@@ -46,18 +46,18 @@ onMount(async () => {
 });
 </script>
 
-<div class="flex flex-wrap items-center justify-between px-2 h-6 bg-[#302251] text-sm">
-  <div class="w-1/2">
-    <ul class="flex flex-wrap list-none space-x-4">
+<div class="flex items-center justify-between px-1 bg-[#302251] text-sm py-0.5 space-x-2">
+  <div class="-mx-1">
```

Comment:
it looks like a code smell to have `-mx-1` here and `mx-1` on each StatusBarItem

Reply:
Yes, this is intentional. See limitations at the following Tailwind doc: https://tailwindcss.com/docs/space. This adds margins on both sides of each item, then removes it from the container so that the left margin is back to size.

Reply:
it recommends to use gap in that case
> it’s better to use the [gap utilities](https://tailwindcss.com/docs/gap)

Reply:
I would not also withhold the merge

Reply:
Yes, that is cleaner. Same thing using gap utils pushed via new commit.